packages/perseus/src/widgets/categorizer/categorizer.stories.tsx

```diff
@@ -13,3 +15,14 @@ type StoryArgs = Record<any, any>;
 export const Question1 = (args: StoryArgs): React.ReactElement => {
     return <RendererWithDebugUI question={question1} />;
 };
+
+export const AnswerlessCategorizer = (args: StoryArgs): React.ReactElement => {
+    return (
+        <ServerItemRendererWithDebugUI
+            item={generateTestPerseusItem({
+                question: question1,
+            })}
+            startAnswerless={true}
+        />
+    );
+};
```

Comment:
If you land this after https://github.com/Khan/perseus/pull/2352, can you adjust this to be a CSF 3.0 story instead?

Reply:
For sure! I have it in my notes to update the story after I started reviewing that PR :)